p/psad/psad_UBI-8.3.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Package       : psad
+# Version       : 3.0
+# Source repo   : https://github.com/mrash/psad
+# Tested on     : UBI 8.2
+# Script License: GPL-2.0 License  
+# Maintainer    : Nishikant Thorat <nthorat@us.ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+yum install -y git wget vim
```

Comment:
Can we combine all the installations into a single "yum install" command to make the script cleaner?

Reply:
Done